backend/selenium_tests/programme_population/test_households.py

```diff
@@ -0,0 +1,86 @@
+from django.conf import settings
+from django.core.management import call_command
+
+import pytest
+from page_object.programme_population.households import Households
+from page_object.programme_population.households_details import HouseholdsDetails
+
+pytestmark = pytest.mark.django_db(transaction=True)
+
+
+@pytest.fixture
+def create_programs() -> None:
+    call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/core/fixtures/data-selenium.json")
+    call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/program/fixtures/data-cypress.json")
+    return
+
+
+@pytest.fixture
+def add_households() -> None:
+    call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/registration_data/fixtures/data-cypress.json")
+    call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/household/fixtures/documenttype.json")
+    call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/household/fixtures/data-cypress.json")
+    return
```

Comment:
maybe let's move those fixtures to `conftest.py` I believe that we will use them in many other tests